test/MUXControlsTestAppWPF/MUXControlsTestAppWPF.csproj

```diff
@@ -133,10 +133,6 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
       <Link>Microsoft.UI.Xaml\Themes\%(RecursiveDir)%(Filename)%(Extension)</Link>
     </Content>
-    <Content Include="$(MSBuildProjectDirectory)\..\TestAppUtils\Themes\generic.xaml">
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-      <Link>MUXControlsTestAppWPF\themes\%(RecursiveDir)%(Filename)%(Extension)</Link>
-    </Content>
```

Comment:
Is this not needed anymore?

Reply:
Right, it's coming from a shared items and was giving a duplicated project item warning.